tools/postinstall-android.js

```diff
@@ -65,13 +65,16 @@ const pluginRoot = path.join('plugins', 'com.salesforce');
 const libProjectRoot = path.join('plugins', 'com.salesforce', 'src', 'android', 'libs');
 const appProjectRoot = path.join('platforms', 'android');
 
-console.log('Moving Salesforce libraries to the correct location');
+console.log('Moving Salesforce Mobile SDK for Android Gradle workspace to the correct location');
 shelljs.cp('-R', path.join(libProjectRoot, 'mobile_sdk'), appProjectRoot);
```

Comment:
Note that the copy command, with `-R`, will silently ignore the `mobile_sdk` directory if it is not present.  That's the default case if `update.sh` hasn't been run, which sets the app to use the Maven Central MSDK artifacts.